spec/unit/provider/package/rpm_spec.rb

```diff
@@ -7,12 +7,13 @@
 
   let (:packages) do
     <<-RPM_OUTPUT
-    cracklib-dicts 0 2.8.9 3.3 x86_64
+    'cracklib-dicts 0 2.8.9 3.3 x86_64
```

Comment:
So my understanding of this would be that rpm outputs a quoted list. Any chance the trailing quote would appear at the end of the last line, rather than by itself on a following line?